commands/web/vite-serve

```diff
@@ -74,6 +74,14 @@ if [ $CMD = start ]; then
   tmux kill-session -t vite-sess 2>/dev/null
   tmux new -s vite-sess -d
   tmux send "node node_modules/vite/bin/vite.js --port $VITE_PRIMARY_PORT --host" C-m
+
+  # Show DDEV info
```

Comment:
Why add this to `viteserve` specifically? `$DDEV_PROJECT_TYPE` and `$DDEV_PRIMARY_URL` are generic ddev environment variables and aren't really related to the vite-serve command.  This might better be added to `ddev describe` in ddev core.

Reply:
It is about your command to start a vite serve, using `ddev start` this information about the project is showed on complete the command